.github/workflows/wbstack.sync.txt

```diff
@@ -0,0 +1,4 @@
+ M vendor/autoload.php
```

Comment:
Probably makes sense to rename this file now that it isn't an output of the sync command but is instead an expected output of the `git status --porcelain=v1` command

Reply:
Changed to `wbstack.sync.expect.txt`